src/components/application_manager/test/hmi_capabilities_test.cc

```diff
@@ -342,29 +342,6 @@ TEST_F(HMICapabilitiesTest, LoadCapabilitiesFromFile) {
   EXPECT_EQ("SE", vehicle_type_so["trim"].asString());
 }
 
-TEST_F(HMICapabilitiesTest, ConvertJsonLanguagesToObj) {
```

Comment:
@anosach-luxoft 
This interface has been removed so I don't think that we have to test it than.